packages/@react-aria/color/src/useColorArea.ts

```diff
@@ -427,14 +443,17 @@ export function useColorArea(props: AriaColorAreaOptions, state: ColorAreaState)
         but remove aria-hidden to reveal the input for each channel when the value has changed with the keyboard.
       */
       'aria-hidden': (isMobile || !focusedInput || focusedInput === 'x' || valueChangedViaKeyboard ? undefined : 'true'),
-      onChange: (e: ChangeEvent<HTMLInputElement>) => {
-        state.setXValue(parseFloat(e.target.value));
-      }
+      onChange
     },
     yInputProps: {
       ...yInputLabellingProps,
       ...visuallyHiddenProps,
       ...yInputFocusProps,
+      style: {
+        ...visuallyHiddenProps.style,
+        WebkitAppearance: 'slider-vertical'
+      },
+      orient: 'vertical',
```

Comment:
What were these added for? I see a deprecation warning in the chrome devtools for -webkit-appearance, and mdn says orient is also non-standard. Do we need them if we already have `aria-orientation` and these are visually hidden anyway?

Reply:
Trying to provide a better distinction between the `x` and `y` axis when interacting with the ColorArea using a screen reader, per [#4110](https://github.com/adobe/react-spectrum/issues/4110).